crypto/merkle/rfc6962_test.go

```diff
@@ -28,13 +28,20 @@ func TestRFC6962Hasher(t *testing.T) {
 	leafHash := leafHashTrail.Hash
 	_, leafHashTrail = trailsFromByteSlices([][]byte{{}})
 	emptyLeafHash := leafHashTrail.Hash
+	_, emptyHashTrail := trailsFromByteSlices([][]byte{})
+	emptyTreeHash := emptyHashTrail.Hash
 	for _, tc := range []struct {
 		desc string
 		got  []byte
 		want string
 	}{
-		// Since creating a merkle tree of no leaves is unsupported here, we skip
-		// the corresponding trillian test vector.
+		// Check that empty trees return the hash of an empty string.
+		// echo -n '' | sha256sum
+		{
+			desc: "RFC6962 Empty Tree",
+			want: "e3b0c44298fc1c149afbf4c8996fb92427ae41e4649b934ca495991b7852b855"[:tmhash.Size*2],
```

Comment:
why times two in `[:tmhash.Size*2]`.
Matches sha256({}): https://github.com/google/trillian/blob/7166f485ebc965d24116db667c9e058a8ed9275c/merkle/rfc6962/rfc6962_test.go#L39